### PR TITLE
Ch react setup

### PR DESCRIPTION
**Pull request by pextech**
**Description**

     This set up a project from scratch without using npx create-react-app

**Description of Task to be completed**

            -Install and configure Wepack 5 & Babel 7
            -Configured The App to Use TailwindCss
            -Set up React Router and created a dummy route "/" for the landing page.
            -Added a ReadMe file which includes: Installation and run commands

**How should this be manually tested?**

            -Install Node js or make sure You have it already
            -To clone, run https://github.com/atlp-rwanda/elite-bn-frontend.git
            -cd into the project directory
            -Run npm install in the terminal
            -To run the application in development mode, run npm run dev
            -Then open http://localhost:8080/ to view it in the browser.

**What are the relevant Trello story?**

Ch-react-setup >> https://trello.com/c/SI7Gb2nA

**What are Relevant Pictures**

Screen Shot 2021-01-19 at 13 56 45

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "extends": ["@imaginary-cloud/react"]
+
```
Comment (nicele08) [inline .eslintrc]: **setup eslint with AirBnB  as popular style guide**
> - https://www.andrewmin.info/blog/react-setup/

**add rules to configure linebreak-style for windows**
> - eslint-errors

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+dist
+node_modules
```
Comment (nicele08) [inline .gitignore:2]: add `.env`

**Review by nicele08 — CHANGES_REQUESTED** [review]

**Review by NIRoberto — APPROVED** [review]

File: .eslintrc (hunk 1 above)
Comment (pextech) [reply .eslintrc]: Eslint is already configured, thank you

**Review by nicele08 — APPROVED** [review]

**Comment by 444root** [comment]
LGTM

**Review by 444root — APPROVED** [review]

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"]
+  }
```
Comment (b0nbon1) [inline .babelrc]: add new line here

File: postcss.config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+const tailwindcss = require('tailwindcss');
+
+module.exports = {
+  // eslint-disable-next-line global-require
```
Comment (b0nbon1) [inline postcss.config.js]: could you add this file to eslintignore?

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Barefoot Nomad</title>
+</head>
+
+<body>
+    <div id="root"></div>
+</body>
+
+</html>
```
Comment (b0nbon1) [inline public/index.html]: add new line here

File: src/styles.css [hunk 6]
```diff
@@ -0,0 +1,5 @@
+@tailwind base;
+
+@tailwind components;
+
+@tailwind utilities;
```
Comment (b0nbon1) [inline src/styles.css]: add new line at the end

**Review by b0nbon1 — CHANGES_REQUESTED** [review]
JUst implement those minor request and the PR will be ready for merging

**Review by NIRoberto — APPROVED** [review]
LGTM

**Review by b0nbon1 — APPROVED** [review]

**Pull request merged**